tests/lib/rules/no-v-html.js

```diff
@@ -28,6 +28,11 @@ ruleTester.run('no-v-html', rule, {
     {
       filename: 'test.vue',
       code: '<template><div v-if="foo" v-bind="bar"></div></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div v-html="knownHtml"></div></template>',
+      options: [{ignorePattern: '/^(?:html|.+Html)$/'}]
```

Comment:
Please run `npm run update` to fix formatting inconsistencies (like missing spaces between the curly brackets here.